c_glib/configure.ac

```diff
@@ -18,7 +18,10 @@
 AC_PREREQ(2.65)
 
 m4_define([arrow_glib_version], m4_include(version))
-AC_INIT([arrow-glib], arrow_glib_version, [kou@clear-code.com])
+AC_INIT([arrow-glib],
+        arrow_glib_version,
+        [https://issues.apache.org/jira/browse/ARROW],
+        [apache-arrow-glib])
```

Comment:
The 4th argument is package archive name. e.g.: `apache-arrow-glib-0.2.1.tar.gz`